### PR TITLE
Remove old entity endpoints

### PR DESCRIPTION
**Pull request by TimDiekmann**
## 🌟 What is the purpose of this PR?

The get endpoints for entities has been deprecated and can be removed now

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1201095311341924/1203542859987545/f) _(internal)_

## 🚫 Blocked by

- [Use query endpoint over GET endpoints](https://app.asana.com/0/1201095311341924/1202979057056097/f) _(internal)_
- #1681

### CODE REVIEW

File: packages/graph/hash_graph/tests/rest-test.http [hunk 1]
```diff
@@ -671,14 +735,18 @@ Content-Type: application/json
...
 {
-  "entityId": "{{person_b_entity_id}}",
-  "ownedById": "{{account_id}}",
-  "actorId": "{{account_id}}"
+ "actorId": "{{account_id}}",
+ "entityId": "{{person_b_entity_id}}",
+ "entityTypeId": "http://localhost:3000/@alice/types/entity-type/person/v/2",
+ "properties": {
+   "http://localhost:3000/@alice/types/property-type/name/": "Alice Allison"
+ },
```
Comment (TimDiekmann) [inline packages/graph/hash_graph/tests/rest-test.http:748]:
```suggestion
 "entityTypeId": "http://localhost:3000/@alice/types/entity-type/person/v/1",
 "properties": {
   "http://localhost:3000/@alice/types/property-type/name/": "Bob"
 },
```

**Review by thehabbos007 — COMMENTED** [review]
Hey, looks good. I have one question which I think we should address before merging

File: packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs [hunk 2]
```diff
@@ -366,9 +227,6 @@ struct UpdateEntityRequest {
     actor_id: UpdatedById,
     #[serde(flatten)]
     order: EntityLinkOrder,
-    // TODO: Remove default value when exposing temporal versions to backend
-    //   see https://app.asana.com/0/0/1203444301722133/f
-    #[serde(default)]
```
[the remark this answers is not in the document]
Comment (TimDiekmann) [reply packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs:371]: I think this was adjusted as part of https://github.com/hashintel/hash/pull/1681#discussion_r1050190588

**Review by thehabbos007 — APPROVED** [review]
Thanks Tim! Tested locally and it seems to work. Let's assume CI can pick up any usages that weren't migrated to the new endpoints

**Review by thehabbos007 — APPROVED** [review]
Those darn imports!

**Pull request merged**